demo/app.js

```diff
@@ -502,10 +523,38 @@ const Demo = React.createClass({
     });
   },
 
+  _nextSibling () {
+    console.log('next');
+  },
+
+  _previousSibling () {
+    console.log('previous');
```

Comment:
console logs left over in the demo here.  can we kill them, or are they serving a purpose?